{"trial_id": "0000", "hyperparameters": {"space": [{"class_name": "Choice", "config": {"name": "activation", "default": "relu", "conditions": [], "values": ["relu", "gelu", "selu"], "ordered": false}}, {"class_name": "Int", "config": {"name": "first_units", "default": null, "conditions": [], "min_value": 1, "max_value": 256, "step": 7, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "num_layers", "default": null, "conditions": [], "min_value": 1, "max_value": 5, "step": 1, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "units_0", "default": null, "conditions": [], "min_value": 32, "max_value": 256, "step": 16, "sampling": "linear"}}, {"class_name": "Choice", "config": {"name": "learning_rate", "default": 1.0, "conditions": [], "values": [1.0, 0.1, 0.01, 0.001, 0.0001], "ordered": true}}, {"class_name": "Int", "config": {"name": "units_1", "default": null, "conditions": [], "min_value": 32, "max_value": 256, "step": 16, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "units_2", "default": null, "conditions": [], "min_value": 32, "max_value": 256, "step": 16, "sampling": "linear"}}], "values": {"activation": "selu", "first_units": 211, "num_layers": 3, "units_0": 32, "learning_rate": 0.1, "tuner/epochs": 2, "tuner/initial_epoch": 0, "tuner/bracket": 4, "tuner/round": 0, "units_1": 32, "units_2": 32}}, "metrics": {"metrics": {}}, "score": null, "best_step": 0, "status": "FAILED", "message": "Traceback (most recent call last):\n  File \"/opt/anaconda3/lib/python3.9/site-packages/keras_tuner/engine/base_tuner.py\", line 266, in _try_run_and_update_trial\n    self._run_and_update_trial(trial, *fit_args, **fit_kwargs)\n  File \"/opt/anaconda3/lib/python3.9/site-packages/keras_tuner/engine/base_tuner.py\", line 257, in _run_and_update_trial\n    tuner_utils.convert_to_metrics_dict(\n  File \"/opt/anaconda3/lib/python3.9/site-packages/keras_tuner/engine/tuner_utils.py\", line 225, in convert_to_metrics_dict\n    [convert_to_metrics_dict(elem, objective) for elem in results]\n  File \"/opt/anaconda3/lib/python3.9/site-packages/keras_tuner/engine/tuner_utils.py\", line 225, in <listcomp>\n    [convert_to_metrics_dict(elem, objective) for elem in results]\n  File \"/opt/anaconda3/lib/python3.9/site-packages/keras_tuner/engine/tuner_utils.py\", line 238, in convert_to_metrics_dict\n    best_value, _ = _get_best_value_and_best_epoch_from_history(\n  File \"/opt/anaconda3/lib/python3.9/site-packages/keras_tuner/engine/tuner_utils.py\", line 209, in _get_best_value_and_best_epoch_from_history\n    objective_value = objective.get_value(metrics)\n  File \"/opt/anaconda3/lib/python3.9/site-packages/keras_tuner/engine/objective.py\", line 57, in get_value\n    return logs[self.name]\nKeyError: 'val_accuracy'\n"}